tests/core/test_fileio.py

```diff
@@ -171,8 +172,9 @@ def test_save_restore_from_nemo_file(self):
 
         with tempfile.NamedTemporaryFile() as fp:
             filename = fp.name
-            asr_model.save_to(save_path=filename)
-            asr_model2 = EncDecCTCModel.restore_from(restore_path=filename)
+            torch.save(asr_model.state_dict(), filename)
+            asr_model2 = EncDecCTCModel(cfg=modelConfig)
+            asr_model2.load_state_dict(torch.load(filename))
```

Comment:
I mean I'm sure this might work better. But the whole point of this test to make sure our `save_to` and `restore_from` make sense. Just mark it as `pleasefixme` and @okuchaiev can you fix when you get a chance?

Reply:
Yes this sounds like a better plan, ill revert this for now.

Reply:
so, pytorch lightning calls the class object again while restoring weights, because of this our test fails now. Since we already instantiated the object, I wonder why they call the cls again?  https://github.com/PyTorchLightning/pytorch-lightning/blob/7b4db3045dcc9e6bb0b66e409b25bb2c7fa378f0/pytorch_lightning/core/saving.py#L205

Reply:
reverted this patch for the moment

Reply:
wait... load_state_dict() is original PT method...

Reply:
Yes and we have a wrapper around that in `FileIO`, so we want to test our wrappers not the original pytorch functions. Hence why I ask Som to reverse this fix

Reply:
Reverted